cobalt/build/packager.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+"""Packages Cobalt with a given layout."""
+
```

Comment:
Please put a comment here at the top as to where the JSON format comes from. A link to the origin Proto schema from Chromium, and a mention that we only implement a subset.

Also - a small test just to show that `files`, `dirs`, `rename_files` and `rename_dirs` keys are respected.

Other than that LGTM